exercises/game-of-life/canonical-data.json

```diff
@@ -0,0 +1,143 @@
+{
+  "exercise": "game-of-life",
+  "cases": [
+    {
+      "uuid": "ae86ea7d-bd07-4357-90b3-ac7d256bd5c5",
+      "description": "empty matrix",
+      "property": "tick",
+      "input": {
+        "matrix": []
+      },
+      "expected": []
+    },
+    {
+      "uuid": "4ea5ccb7-7b73-4281-954a-bed1b0f139a5",
+      "description": "live cells with zero live neighbors die",
+      "property": "tick",
+      "input": {
+        "matrix": [
+          [0, 0, 0],
+          [0, 1, 0],
+          [0, 0, 0]
+        ]
+      },
+      "expected": [
+        [0, 0, 0],
+        [0, 0, 0],
+        [0, 0, 0]
+      ]
+    },
+    {
+      "uuid": "df245adc-14ff-4f9c-b2ae-f465ef5321b2",
+      "description": "live cells with only one live neighbor die",
+      "property": "tick",
+      "input": {
+        "matrix": [
+          [0, 0, 0],
+          [0, 1, 0],
+          [0, 1, 0]
+        ]
+      },
+      "expected": [
+        [0, 0, 0],
+        [0, 0, 0],
+        [0, 0, 0]
+      ]
+    },
+    {
+      "uuid": "2a713b56-283c-48c8-adae-1d21306c80ae",
+      "description": "live cells with two live neighbors stay alive",
+      "property": "tick",
+      "input": {
+        "matrix": [
+          [0, 1, 0],
+          [1, 0, 0],
+          [0, 1, 0]
+        ]
+      },
+      "expected": [
+        [0, 0, 0],
+        [1, 1, 0],
+        [0, 0, 0]
+      ]
```

Comment:
This test jumps ahead to "dead cells with three live neighbors become alive". Would it be better to use this input for testing "live cells with two live neighbors stay alive":
```
      "input": {
        "matrix": [
          [1, 0, 0],
          [1, 0, 0],
          [1, 0, 0]
        ]
      },
      "expected": [
        [0, 0, 0],
        [1, 0, 0],
        [0, 0, 0]
      ]
```
or even
```
      "input": {
        "matrix": [
          [1, 0, 1],
          [1, 0, 1],
          [1, 0, 1]
        ]
      },
      "expected": [
        [0, 0, 0],
        [1, 0, 1],
        [0, 0, 0]
      ]
```

Reply:
On your first test suggestion, there is still a dead cell with three live neighbors. But  the second one is great, will update